torchtext/vocab/vocab_factory.py

```diff
@@ -6,7 +6,7 @@
 )
 
 
-def vocab(ordered_dict: Dict, min_freq: int = 1) -> Vocab:
+def vocab(ordered_dict: Dict, min_freq: int = 1, specials: List[str] = [], special_first: bool = True) -> Vocab:
```

Comment:
I think we may want to use None as default value of mutable arguments per the [docs](https://docs.python.org/3/reference/compound_stmts.html#function-definitions). So instead we could do:
```
specials: Optional[List[str]] = None
```

Reply:
Sounds reasonable. I'll refactor this!

Reply:
Done!